### ISSUE TITLE
Which limit is correct for DurableObjects (req/s)

### ISSUE THREAD

**Issue by ambrt** [cloudflare/cloudflare-docs#10550]
### Which Cloudflare product does this pertain to?

Durable objects

### Existing documentation URL(s)

https://developers.cloudflare.com/durable-objects/platform/limits/

https://developers.cloudflare.com/workers/platform/limits/#durable-objects

### What changes are you suggesting?

There are two different limits for DurableObjects - requests per second. 
In [Workers docs](https://developers.cloudflare.com/workers/platform/limits/#durable-objects) limit is 100req/s and in [Durable objects docs](https://developers.cloudflare.com/durable-objects/platform/limits/) it's 1,000 req/s.

Which one  is correct?

**Comment by helloimalastair** [comment]
Neither, *kind of*. Both of them are soft limits, in the sense that there isn't a part of the Workers runtime that explicitly limits you to 1k requests per second. Instead, the 1k (and 100) rps limits are based on observations that, *on average*, a Durable Object will be able to handle that many requests per second before being unable to keep up. The 100 req/s limit was the old one, and the 1k is the one updated more recently, and thus should be more accurate.

TL;DR: Soft Limit is 1k rps, but you may be able to optimize to get higher than that.

**Comment by ambrt** [comment]
@helloimalastair I will keep this issue opened since i think docs should be edited to reflect what you wrote.  Thank you for explanation.

**Comment by kodster28** [comment]
I believe this has been addressed by #10577
